tests/sys/fido2_ctap/Makefile

```diff
@@ -1,39 +1,19 @@
-BOARD ?= nrf52840dk
-#BOARD ?= nrf52840dongle
+BOARD ?= native
 
 include ../Makefile.sys_common
 
-USEMODULE += fido2_ctap_transport_hid
-USEMODULE += usbus
-USEMODULE += ztimer_sec
-USEPKG += fido2_tests
+# same as CTAP_STACKSIZE
+CFLAGS += -DTHREAD_STACKSIZE_MAIN=15000
```

Comment:
this looks strange to me would assume  a 2^n or multiple of 1024 here

but i grepped trough RIOT and there are some other cases like this

Reply:
This was just something I measured when testing fido2. I think it was not 16k back then but maybe 4k ? Thats why I explicitly set a bigger stack size. I can change this to be 16 though